sotodlib/coords/local.py

```diff
@@ -0,0 +1,458 @@
+import numpy as np
+from astropy import units as u
+
+
+'''
+Most of the functions are adapted from pymap3d, to include error calulations 
+'''
```

Comment:
Please use double quotes for comment blocks.  """/"""

Typo "calculations"

Explain purpose of this submodule.  For example, the fact that these functions are important for drone observations / simulations! 

Maybe mention enu/geodetic/ecef coord systems with a brief line about each one?  We can have all these docs appear on the readthedocs page automagically.

Reply:
Done

Reply:
That's very helpful, thanks.